HelpMyStreetFE/HelpMyStreetFE/src/sass/components/profile/_job.scss

```diff
@@ -11,181 +11,193 @@
 }
 
 .job {
-  @include shadow-xs;
-  width: 100%;
-  min-height: 50px;
-  border: 1px solid $gray-8;
-  border-radius: 5px;
-  margin: 10px 0;
-  padding: 5px;
-  overflow: hidden;
-
-  @media (min-width: $med-screen) {
-    padding: 10px 5px;
-  }
-
-  &.highlight {
-    animation: highlight 2s;
-  }
-
-  &__info {
-    min-height: 100px;
-    display: grid;
-    grid-template-columns: 1fr;
+    @include shadow-xs;
+    width: 100%;
+    min-height: 50px;
+    border: 1px solid $gray-8;
+    border-radius: 5px;
+    margin: 10px 0;
+    padding: 5px;
```

Comment:
Github is showing alot of changes here, but all i did int he SCSS file was 

1.) Change 
  @media (min-width: $med-screen) {
            grid-template-columns: 1fr 2fr 1fr;
        }
to
  @media (min-width: $med-screen) {
            grid-template-columns: 1fr 2fr 1.3fr;
        }

2. added the below class to &__actions > .btn 
   &.accepted {
                    background: none;
                    border: none;
                    color: $green;
                }
3.) added     the below class to &__footer
  &.accepted{
                display:none;
            }